app/views/shared/_modal_layout.html.erb

```diff
@@ -1,9 +1,13 @@
 <div class="modal-backdrop">
 </div>
-<div class="padding-x-2 padding-y-6 modal" role="dialog">
-  <div class="modal-center">
-    <div class="modal-content">
-      <%= yield %>
+  <%= tag.div(role: 'dialog', 
+      class: 'padding-x-2 padding-y-6 modal', 
+      aria: { describedby: local_assigns[:description_id], 
+              labelledby: local_assigns[:label_id] }) do %>
+    <div class="modal-center">
+      <div class="modal-content">
+        <%= yield %>
+      </div>
```

Comment:
Hi sorry to be jumping in late on this PR, but I had an idea... I think we could generate random values for the label id & description_id, and then yield those to the block:

```suggestion
  <%= tag.div(role: 'dialog', 
      class: 'padding-x-2 padding-y-6 modal', 
      aria: { describedby: (description_id = SecureRandom.uuid), 
              labelledby: (label_id = SecureRandom.uuid) }) do %>
    <div class="modal-center">
      <div class="modal-content">
        <%= yield label_id, description_id %>
      </div>
```

So using it in another location would look like:

```
<%= render layout: '/shared/modal_layout' do |label_id, description_id| %>
  ...
  <h2 class="margin-y-2 fs-20p sans-serif regular center" id='<%= label_id %>'>
  ...
  <div class="margin-bottom-8" id="<%= description_id %>">
```

Just an idea